doc/couscous.yml

```diff
@@ -0,0 +1,70 @@
+title: Gaufrette
+subTitle: Filesystem abstraction layer
+github:
+    user: KnpLabs
+    repo: Gaufrette
+baseUrl: http://knplabs.github.io/Gaufrette
```

Comment:
you may use the HTTPS base url here (btw, couscous is currently not friendly to websites available both as HTTP and HTTPS if you use HTTP in the base url, as it will load mixed content, so using the HTTPS url would change things a lot for people loading all github.io pages in HTTPS because of HTTPS Everywhere or similar extensions)